utils/macos/build_app.sh

```diff
@@ -77,13 +77,15 @@ if [ ! -d "$SDK_DIRECTORY" ]; then
    fi
 fi
 
-REVISION=`bzr revno $SOURCE_DIR`
+REVISION=`git rev-parse --short=5 HEAD`[`git --git-dir=$SOURCE_DIR/.git rev-parse --abbrev-ref HEAD`]
 DESTINATION="WidelandsRelease"
 
 if [[ -f $SOURCE_DIR/WL_RELEASE ]]; then
    WLVERSION="$(cat $SOURCE_DIR/WL_RELEASE)"
+   COMMIT_COUNT=$WLVERSION
 else
-   WLVERSION="r$REVISION"
+   WLVERSION="git-$REVISION"
```

Comment:
`git-r12354` is not unique. You have to mention the SHA somehow. see above for my suggestion, here simply `$REVISION`

Reply:
Remember to remove the git-